docs/UserGuide.md

```diff
@@ -442,6 +448,7 @@ if you are currently in the `plane` mode, then this command will link a `plane`
 <div style="border: 0px solid #ccc; background-color: #d9edff; color: darkblue; padding: 10px; margin-bottom: 10px;">
 Note: The prefixes vary across different modes. Refer to the examples below for more clarity.
 </div>
+Here are some examples of how the command works in each mode:
```

Comment:
This line can be removed since its similar to the one in the callout just above it